test/auth.js

```diff
@@ -246,7 +253,7 @@ test('core using custom sign fn', async function (t) {
   t.is(b.length, 5)
 })
 
-function hash (...data) {
+function cryptoHash (...data) {
   const out = b4a.alloc(32)
   sodium.crypto_generichash(out, b4a.concat(data))
```

Comment:
Sounds like we can remove this helper in favor of `const out = crypto.hash(data)` from hypercore-crypto that we already have installed

Reply:
Unless we need that `alloc` instead of `allocUnsafe` but it should be ok